Makefile.core.mk

```diff
@@ -109,7 +109,8 @@ fixup_generated_files=\
 .PHONY: generate-k8s-client
 generate-k8s-client:
 	# generate kube api type wrappers for istio types
-	@KUBETYPE_GOLANG_PROTOBUF=true $(kubetype_gen) --input-dirs $(kube_istio_source_packages) --output-package $(kube_api_base_package) -h $(kube_go_header_text)
+	# TODO(https://github.com/istio/istio/issues/54831) do not depend on gotypesalias=0 for this to work
+	@GODEBUG=gotypesalias=0 $(kubetype_gen) --input-dirs $(kube_istio_source_packages) --output-package $(kube_api_base_package) -h $(kube_go_header_text)
```

Comment:
This is the real fix, rest was just updating deps/common-files since its been backed up for so long